src/EditorFeatures/Core/Implementation/IntelliSense/Completion/AsyncCompletionService.cs

```diff
@@ -30,6 +30,14 @@ internal class AsyncCompletionService : ForegroundThreadAffinitizedObject, IAsyn
         private readonly IAsynchronousOperationListener _listener;
         private readonly IEnumerable<Lazy<IBraceCompletionSessionProvider, BraceCompletionMetadata>> _autoBraceCompletionChars;
         private readonly Dictionary<IContentType, ImmutableHashSet<char>> _autoBraceCompletionCharSet;
+        private readonly IFeatureCookie _legacyCompletionEnabledCookie;
+
+#pragma warning disable IDE1006 // Naming Styles -- This name is picked up by the IFeatureService.
```

Comment:
❔  Why not make it a property, or a `public static readonly` field? Both cases have PascalCase naming styles.

Reply:
It's a MEF part and therefore cannot be static.

Reply:
But a property would still work?